docs/howto/logging.txt

```diff
@@ -0,0 +1,337 @@
+.. _logging-how-to:
+
+================================
+How to configure and use logging
+================================
+
+.. seealso::
+
+    * :ref:`Django logging reference <logging_ref>`
+    * :ref:`Django logging overview <logging-explanation>`
+
+Django provides a working :ref:`default logging configuration
+<logging-default-configuration>` that is readily extended.
+
+Make a basic logging call
+=========================
+
+To send a log message from within your code, you place a logging call into it.
+
+.. admonition:: Don't be tempted to use logging calls in ``settings.py``.
+
+    The way that Django logging is configured as part of the ``setup()``
+    function means that logging calls placed in ``settings.py`` may not work as
+    expected, because *logging will not be set up at that point*. To explore
+    logging, use a view function as suggested in the example below.
+
+First, import the Python logging library, and then obtain a logger instance
+with :py:func:`logging.getLogger`. The ``getLogger()`` method must be provided
```

Comment:
`must be provided` -> `should be provided` _maybe_? `logging.getLogger()` without a name returns the root logger.